DataTypes.mw.php

```diff
@@ -32,7 +32,7 @@
 	$args = func_get_args();
 	$key = array_shift( $args );
 	$language = array_shift( $args );
-	$message = new \Message( $key, $args );
```

Comment:
Oh. Thanks, fixed it. That's why I do not like atomic class names like "Query" or "Message".